client/package.json

```diff
@@ -18,17 +18,16 @@
 	"typings": "./lib/main",
 	"devDependencies": {
 		"typescript": "^2.1.5",
-		"vscode": "^1.0.0",
 		"@types/node": "^6.0.42"
 	},
 	"dependencies": {
 		"@sourcegraph/vscode-jsonrpc": "^3.0.4-patch",
-		"vscode-languageserver-types": "^3.0.3"
+		"@sourcegraph/vscode-languageserver-types": "^3.0.3-patch"
 	},
 	"scripts": {
-		"prepublish": "node ./node_modules/vscode/bin/install && tsc -p ./src",
+		"prepublish": "tsc -p ./src",
```

Comment:
It is impossible to compile this with vanilla VS code due to our proposed API changes. The only way to develop against this repo is to have these changes and have a local copy of vscode, so I was thinking I might as well check them in.